app/routes/events/view/tickets/discount-codes/create.js

```diff
@@ -8,26 +7,6 @@ export default Route.extend({
     return this.l10n.t('Create');
   },
   model() {
-    return {
-      code          : '',
-      isActive      : true,
-      ticketsNumber : 10,
-      minQuantity   : 1,
-      maxQuantity   : 10,
-      validFromDate : moment(),
-      validFromTime : '12:00',
-      validTillDate : moment().add(20, 'days'),
-      validTillTime : '16:00',
-      tickets       : [
-        {
-          name : 'Early bird',
-          id   : 1
-        },
-        {
-          name : 'Late entry',
-          id   : 2
-        }
-      ]
-    };
+    return this.get('store').findAll('discountCode');
```

Comment:
@harshitagupta30 We are creating a discount code not listing the codes. Please check how we are doing it for other routes